snake.cpp

```diff
@@ -0,0 +1,99 @@
+#include"bits/stdc++.h"
+using namespace std;
+#define fast ios::sync_with_stdio(0); cin.tie(0); cout.tie(0);
+#define pb push_back
+#define mp make_pair
+#define int long long
+const int mm = 1e5 + 5;
+/* space = 1 , tab = 2 for better understanding sublime test editor*/
+unordered_map<int, int>snakeH;
+unordered_map<int, int>snakeT;
+unordered_map<int, int>ladderH;
+unordered_map<int, int>ladderT;
+
+class games_in{
```

Comment:
When you are creating a class it should represent some enity for eg in this case Board, Ladder etc so try to think in more OOP way and refactor the code.